<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>420</width>
                <height>294</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>420</width>
                <height>294</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>420</width>
                <height>294</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>SmartDesk - Desktop erstellen</string>
        </property>
        <property name="styleSheet">
            <string notr="true">#Form {
                background-color: #2b2b2b;
                border-radius: 10px;
                }

                QWidget {
                color: #ffffff;
                }

                QLabel {
                color: #cccccc;
                }

                QLineEdit {
                background-color: #3c3c3c;
                color: #ffffff;
                border: 1px solid #555555;
                border-radius: 4px;
                padding: 5px;
                }

                QLineEdit:focus {
                border: 1px solid #14a085;
                }

                QPushButton {
                background-color: #3c3c3c;
                color: #ffffff;
                border: none;
                border-radius: 4px;
                padding: 8px;
                }

                QPushButton:hover {
                background-color: #4a4a4a;
                }

                QPushButton:pressed {
                background-color: #2a2a2a;
                }

                QPushButton#btn_create {
                background-color: #14a085;
                color: #ffffff;
                }

                QPushButton#btn_create:hover {
                background-color: #0d7377;
                }

                QPushButton#btn_create:pressed {
                background-color: #0a5a5d;
                }

                QRadioButton {
                color: #ffffff;
                spacing: 8px;
                }

                QRadioButton::indicator {
                width: 16px;
                height: 16px;
                }

                QRadioButton::indicator:unchecked {
                border: 2px solid #555555;
                border-radius: 8px;
                background-color: #3c3c3c;
                }

                QRadioButton::indicator:checked {
                border: 2px solid #14a085;
                border-radius: 8px;
                background-color: #14a085;
                }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
                <number>8</number>
            </property>
            <property name="leftMargin">
                <number>20</number>
            </property>
            <property name="topMargin">
                <number>15</number>
            </property>
            <property name="rightMargin">
                <number>20</number>
            </property>
            <property name="bottomMargin">
                <number>15</number>
            </property>
            <item>
                <widget class="QLabel" name="label_title">
                    <property name="font">
                        <font>
                            <family>Segoe UI</family>
                            <pointsize>14</pointsize>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Desktop erstellen</string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer_top">
                    <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Policy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>3</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QVBoxLayout" name="layout_name_field">
                    <property name="spacing">
                        <number>4</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label_name">
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Name</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineEdit_name">
                            <property name="minimumSize">
                                <size>
                                    <width>0</width>
                                    <height>32</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="placeholderText">
                                <string>Desktop-Name eingeben...</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QVBoxLayout" name="layout_path_field">
                    <property name="spacing">
                        <number>4</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label_path">
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Pfad zum vorhandenen Ordner:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="layout_path_input">
                            <property name="spacing">
                                <number>8</number>
                            </property>
                            <item>
                                <widget class="QLineEdit" name="lineEdit_path">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>32</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <family>Segoe UI</family>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="placeholderText">
                                        <string>Ordner ausw√§hlen...</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_browse">
                                    <property name="minimumSize">
                                        <size>
                                            <width>40</width>
                                            <height>32</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>40</width>
                                            <height>32</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <family>Segoe UI</family>
                                            <pointsize>10</pointsize>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>...</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="layout_radio_buttons">
                    <property name="spacing">
                        <number>15</number>
                    </property>
                    <item>
                        <widget class="QRadioButton" name="radioButton_existing">
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Vorhanden</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QRadioButton" name="radioButton_new">
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Neu erstellen</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_radio">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer_bottom">
                    <property name="orientation">
                        <enum>Qt::Orientation::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="layout_action_buttons">
                    <property name="spacing">
                        <number>10</number>
                    </property>
                    <item>
                        <spacer name="horizontalSpacer_buttons">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_create">
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>36</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                    <bold>true</bold>
                                </font>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="styleSheet">
                                <string notr="true" />
                            </property>
                            <property name="text">
                                <string>Erstellen</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_cancel">
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>36</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <family>Segoe UI</family>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="text">
                                <string>Abbrechen</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>
